modules/rhel-compute-using-hooks.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-rhel-compute.adoc
+
+[id="rhel-compute-using-hooks-{context}"]
+= Configuring the Ansible inventory file to use hooks
+
+You define the hooks to use when you update the Red Hat Enterprise Linux (RHEL)
+compute, or worker, machines in the `hosts` inventory file under the `OSEv3:vars`
+section.
+
+.Prerequisites
+
+* You have access to the machine that you used to add the RHEL compute machines
+cluster. You must have access to the `hosts` Ansible inventory file that defines
+your RHEL machines.
+
+
+.Procedure
+
+. After you design the hook, create a YAML file that defines the Ansible tasks
+for it. This file must be a set of tasks and cannot be a playbook, as shown in
+the following example:
++
+[source.yaml]
+----
+---
+# Trivial example forcing an operator to acknowledge the start of an upgrade
+# file=/usr/share/custom/pre_compute.yml
+
+- name: note the start of a compute machine update
+  debug:
+      msg: "Compute machine upgrade of {{ inventory_hostname }} is about to start"
+
+- name: require the user agree to start an upgrade
+  pause:
+      prompt: "Press Enter to start the compute machine update"
+----
+
+. Modify the `hosts` Ansible inventory file to specify the hook files. The
+hook files are specified as parameter values in the the `[OSEv3:vars]` section,
+as shown:
++
+.Example hook definitions in an inventory file
+[source]
+----
+[OSEv3:vars]
+openshift_node_upgrade_pre_hook=/usr/share/custom/pre_node.yml
```

Comment:
docs of OCP 4 hooks for reference.
https://github.com/openshift/openshift-ansible/blob/release-4.1/HOOKS.md